tribomobile/App.js

```diff
@@ -7,34 +7,21 @@
  */
 
 import React from 'react';
-
-import {
-  SafeAreaView,
-  Text,
-  StatusBar,
-  View,
-} from 'react-native';
-import titlesText from './src/titlesText';
 import ModalMyAccount from './components/modals/ModalMyAccount';
-
 import {SafeAreaView, Text, StatusBar} from 'react-native';
 import ModalInfoStore from './components/modals/ModalInfoStore';
 import LittlePin from './components/modals/LittlePinInfo';
+import titlesText from './src/titlesText';
+import contentText from './screensText/contentText';
```

Comment:
good practices says we must use upper-case naming format for all components.

```suggestion
import TitlesText from './src/TitlesText';
import ContentText from './screensText/ContentText';
```